lib/brasil_api/pix.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module BrasilAPI
+  class PIX < Base
+    class << self
+      def get_participants
+        get("/pix/v1/participants")
+      end
+    end
+  end
+end
```

Comment:
Adicionar a linha em branco no fim do arquivo:
```suggestion
end

```